numba/core/config.py

```diff
@@ -306,6 +306,11 @@ def avx_default():
         DISABLE_JIT = _readenv("NUMBA_DISABLE_JIT", int, 0)
 
         # choose parallel backend to use
+        THREADING_LAYER_PRIORITY = _readenv(
+            "NUMBA_THREADING_LAYER_PRIORITY",
+            lambda string: string.split(),
+            ['tbb', 'omp', 'workqueue'],
+        )
```

Comment:
```suggestion
        )

```

Reply:
It's adding a new line to match the style of the rest of the variable declarations, for consistency I think it should be present. I also note on the diff that the comments associated with these variables have gone awry, I'll pick up those in a new review as a suggested change.